trento-docs-site/antora.yml

```diff
@@ -8,153 +8,161 @@ nav:
+        - command: git clone --depth 1 --single-branch https://github.com/trento-project/workbench trento-docs-site/build/tmp_components/workbench
 
+    # Generate navigation for components in nav_developer.adoc
+    - run:
+      - command: node trento-docs-site/scripts/generate-components.nav.js
+    - scan:
+      - dir: trento-docs-site/build/gen_navigation
+        files: 'nav_components.adoc'
+        into: modules/developer/partials
     # Trento official documentation repo https://github.com/SUSE/doc-unversioned
     - scan:
-      - dir: trento-docs-site/tmp_docs/user-guide/trento/adoc
+      - dir: trento-docs-site/build/tmp_docs/user-guide/trento/adoc
         files: '**/*.adoc'
         into: modules/user-guide/pages
-      - dir: trento-docs-site/tmp_docs/user-guide/trento/images/src/png
+      - dir: trento-docs-site/build/tmp_docs/user-guide/trento/images/src/png
         files: '**/*.png'
         into: modules/user-guide/images
     - run:
-        command: trento-docs-site/scripts/generate-attributes-yaml.sh ./trento-docs-site/tmp_docs/user-guide/trento/adoc/generic-attributes.adoc ./trento-docs-site/tmp_docs/user-guide/antora_tmp/antora.yml
+        command: trento-docs-site/scripts/generate-attributes-yaml.sh ./trento-docs-site/build/tmp_docs/user-guide/trento/adoc/generic-attributes.adoc ./trento-docs-site/build/adoc_variables/user-guide/antora.yml
     - scan:
-        dir: trento-docs-site/tmp_docs/user-guide/antora_tmp
+        dir: trento-docs-site/build/adoc_variables/user-guide/
         files: 'antora.yml'
     # Trento upstream projects documentation
     ## Trento docs repo https://github.com/trento-project/docs
     - scan:
-      - dir: ./../
+      - dir: ./..
         files: 'README.adoc'
         into: modules/developer/pages
-      - dir:  content/
+      - dir: content
         files: '**/*.adoc'
         into: modules/developer/pages/
-      - dir: content/images/
+      - dir: content/images
         files: '**/*.{png,jpg,jpeg,svg,gif}'
         into: modules/developer/images/images
-    ## Trento Web https://github.com/trento-project/web
-    - scan:
-      - dir: trento-docs-site/tmp_docs/web/
-        files: '*.adoc'
-        into: modules/web/pages
-      - dir: trento-docs-site/tmp_docs/web/guides
-        files: '**/*.adoc'
-        into: modules/web/pages/
-      - dir: trento-docs-site/tmp_docs/web/guides/
-        files: '**/*.{png,jpg,jpeg,svg,gif}'
-        into: modules/web/images/
     ## Trento Agent https://github.com/trento-project/agent
     - scan:
-      - dir: trento-docs-site/tmp_docs/agent/
+      - dir: trento-docs-site/build/tmp_components/agent
         files: '*.adoc'
         into: modules/agent/pages
-      - dir: trento-docs-site/tmp_docs/agent/docs
-        files: '**/*.adoc'
+      - dir: trento-docs-site/build/tmp_components/agent/docs
+        files: ['**/*.adoc', '!README.adoc']
```

Comment:
This forbids the antora collector to fetch the README from /docs. The script also handles this case, it is just to be double sure!